docs/resources/volume_set.md

```diff
@@ -35,6 +35,9 @@ Optional:
 
 - **create_final_snapshot** (Boolean) If true, a volume snapshot will be created immediately before deletion of any volume in this set. Default: `true`
 - **retention_duration** (String) The default retention period for volume snapshots. This string should contain a floating point number followed by either d, h, or m. For example, "10d" would retain snapshots for 10 days.
+- **schedule** (String) A standard cron schedule expression used to determine when your job should execute. (i.e., `0 * * * *` Every hour).
```

Comment:
This should say 

(String) A standard cron schedule expression used to determine when a snapshot will be taken. (i.e., `0 * * * *` Every hour). Note: snapshots cannot be scheduled more often than once per hour.

Reply:
Fixed.